third_party/2and3/redis/client.pyi

```diff
@@ -221,7 +221,10 @@ class Redis(object):
     def smembers(self, name: _Key) -> Set[_Value]: ...
     def smove(self, src, dst, value): ...
     def spop(self, name, count: Optional[int] = ...): ...
-    def srandmember(self, name, number=...): ...
+    @overload
+    def srandmember(self, name: _Key, number: None) -> Optional[_Value]: ...
+    @overload
+    def srandmember(self, name: _Key, number: int) -> List[_Value]: ...
```

Comment:
One little thing and then this is ready to go. The first overload needs `number: None = ...` as `srandmember()` has a default (`None`) for the second argument.